### ISSUE TITLE
[WIP] [Discussion] Reestruturação

### ISSUE THREAD

**Issue by pauloelr** [PHPSP/phpsp-blog#14]
# Motivação

A algum tempo atras eu enviei pelo e-mail do PHPSP um PDF com uma [Proposta de Reestruturação das Categorias do Blog](https://drive.google.com/a/phpsp.org.br/file/d/0B4zif-ZAmbY0NTg1OWtxTmpRd1k/edit?usp=sharing]) na época um pouco foi discutido a respeito, porem não muito foi implementado, essa semana a discussão foi reacendida na Issue #8 e alguns novos pontos foram levantados e estão descritos abaixo
## Mudanças
1. Passamos a realizar gravações de hangouts de forma mais periódica.
2. Mudamos o gerenciamento de eventos para o Meetup
3. Novos braços passaram a integrar o grupo (PHP Campinas, etc)
## Problemas Persistentes
1. Baixo nível de atualização de conteúdo
2. Problemas com atualizações da plataforma em produção
3. Baixa integração com as redes sociais
4. Área em excesso no blog (descrito no PDF)
## Plano de Ações

Sabendo desses pontos e também sabendo da possível existência de outros ainda não discutidos, gostaria de sugerir algumas ações com o objetivo de solucionar os problemas supra-citados.
1. Tempo para os demais integrantes do grupo analisarem e discutirem o que aqui está sendo proposto. 
2. Criação de um novo tema com as categorias reestruturadas e novas áreas que permitam uma integração maior com as redes sociais e divulgação dos vídeos produzidos por todos os braços do grupo em novo repositório.
   - Isso ajudaria a resolver o problema de baixo nível de atualizações do blog, colocaria em prática a reestruturação das categorias, fomentaria a aproximação dos diversos braços do grupo e ajudaria a resolver o problema de atualizações da plataforma em servidor de produção já que o novo repositório abrigaria apenas o novo tema do blog que poderia então ser atualizado de forma independente do restante da aplicação
3. Integração dos eventos com o Meetup
   - Procurei por algum plugin de Wordpress que já resolvesse esse problema de integração e não achei nenhum que fosse completo o suficiente, se alguém souber de algum sugestões são bem vindas
   - Caso nenhum plugin fosse encontrado poderíamos criar um próprio que consultaria a API do Meetup e exibiria os próximos eventos na página, também em um repositório diferentes para permitir que as atualizações nos componentes do blog sejam todas independentes.
## Conclusão

Dito isso, seria bom ouvir a opinião de todos os membros do grupo para que possamos avaliar a melhor forma de ação.

PS: Antes que alguém pergunte "Why so Serious?", é porque a parada é seria mesmo $%#&@$.... \o/

**Comment by casimiroarruda** [comment]
Se possível a todos, poderíamos fazer uma reunião presencial - não um evento do grupo, mas uma reunião de organização. Tomar as decisões e já eleger os responsáveis por tocar cada uma das mudanças - acredito que com essa "nomeação" role inclusive uma motivação para fazer as coisas.

Podemos usar o espaço para comunidades no InterCon para divulgar o trabalho e de fato realizar algumas mudanças lá - por experiências anteriores já vimos que um dia é pouco para realizar grandes coisas - mas se pudermos quebrar o esforço total em pequenos marcos podemos sim realizar muitas coisas lá no dia.

Se desejarem podemos nos organizar (eu penso que isso precisa de uma data limite) em até 2 semanas para essa reunião de estruturação onde montaremos o backlog do nosso "projeto" - Todos os membros ativos (evangelistas e ativistas) são convidados - como sugestão poderíamos dessa vez não fazer em uma empresa - pode ser aqui em casa para ser meio do caminho para o pessoal de campinas. Ou podemos fazer em SP em um local neutro.

O que acham?

**Comment by pauloelr** [comment]
Por mim pode ser assim mesmo, 

Agente se reúne, defini o que tem que ser feito, quem puder se compromete com cada uma das tarefas, da uma estimativa de tempo para realizar essa tarefa, fechamos o backlog e trabalhamos com base nele.

Agora o lugar onde agente vai se reunir pra mim é bastante flexível, então é só ver a disponibilidade de todo mundo e marcar. 

Só não entendi o que você falou do espaço de comunidades no InterCon, você diz tipo aquelas salas quem tiveram no InterCon WP?

**Comment by augustohp** [comment]
Concordo com a restruturação das categorias, eu aproveitaria suas idéias e conceitos e já aceitaria qualquer proposta sua.

Se você tiver as devidas permissões dentro do blog, sou da opinião de que você já pode mudar a parada toda e na "reunião" vemos os impedimentos pra conclusão e coisa e tal.

**Comment by rogeriopradoj** [comment]
:+1:

**Comment by jpjoao** [comment]
Concordo plenamente com a "RFC" proposta.

Assim que a PR 23 (https://github.com/PHPSP/phpsp-blog-theme/pull/23) introduz uma forma mais simples de gerenciar os dois menus no cabeçalho. Já é um começo.
Acredito que o segundo passo seja alterar em massa os conteúdos existentes para a categoria "artigos" como o sugerido. Isso, por si só, iria reduzir pela metade (aproximadamente) a quantidade de taxonomias existentes. O próximo passo seria fazer o mesmo para eventos (aparentemente o segundo mais taxonomico dos itens).

O PR 24 (https://github.com/PHPSP/phpsp-blog-theme/pull/24) adiciona o plugin do meetup, o que já poderia ser o gerenciador de eventos. Ou ao menos uma forma de simplificar sua lógica.

**Comment by jpjoao** [comment]
Qual vai ser o plano de ação aqui? Votação pela adoção da "RFC"? Aqui ou no Slack? Qual data limite?
